_posts/2019-03-21-timezone-in-rails-and-javascript.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title:  "Timezone in Rails and JavaScript"
+date:   2019-03-21 4:00:00 -0400
+categories: Ruby on Rails, JavaScript, timezone, Date
+author: Jason Ji
+---
+
+This blog contains two parts:
+  1. Timezone in Rails App
+  2. Timezone in the browser
+
+-----
+<br>
+
+## Timezone in Rails
+<div class="information information-tip">
+  Rails provides great tools for working with time zones but there's still a lot of things that can go wrong. This talk aims to shed some light on these gotchas and provide solutions to the most common problems.
```

Comment:
`this talk` -> "this blog post" might work better

Reply:
😂YEP